### DISCUSSION TITLE
"In" out Linkdin  - Link to social

### DISCUSSION THREAD

**Discussion by fabiohcnobre** [community/community#52086] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Linkedin in was ugly. Can you take that "in" out?

![image](https://user-images.githubusercontent.com/22354660/230145016-8f2e730b-6a8a-4639-853d-fa0821012bc7.png)

**Comment by iamdulanga** [comment]
**It was a prefix it cannot be removed... i think..**

**Comment by CodeByAshuu** [comment]
Unfortunately, it's not possible to remove the "in" from the LinkedIn URL on your GitHub profile and still have it redirect users correctly to your LinkedIn profile. The "in" is a necessary part of LinkedIn's URL structure to identify individual profiles. So, in this case, it's not feasible to remove the "in" and still maintain a functional link to your LinkedIn profile.
